rust/psibase/src/services/symbol.rs

```diff
@@ -0,0 +1,95 @@
+use async_graphql::{InputObject, SimpleObject};
+use fracpack::{Pack, ToSchema, Unpack};
+use serde::{Deserialize, Serialize};
+
+use crate::{AccountNumber, Quantity};
+
+#[derive(
+    Debug, Copy, Clone, Pack, Unpack, ToSchema, Serialize, Deserialize, SimpleObject, InputObject,
+)]
+#[fracpack(fracpack_mod = "fracpack")]
+pub struct SaleDetails {
+    #[allow(non_snake_case)]
```

Comment:
Putting this on the field doesn't seem to work. I'm getting a flood of warnings (rust 1.87)

Reply:
I tried completely removing all the derives, and still got a warning.

Reply:
Is it still persisting? I get no warnings.